alcs/acs2/Classifier.py

```diff
@@ -326,9 +335,11 @@ def is_similar(self, other) -> bool:
                 self.action == other.action and \
                 self.effect == other.effect:
             return True
-
         return False
 
+    def is_equally_general(self, other) -> bool:
```

Comment:
Is this method used somewhere?

Reply:
Yes, in `ClassifiersList.find_subsumer()`